OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -102,90 +95,304 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    The board attachment is uncommon. It only consists of two holes drilled through the thickness of the boards.
                                 </decoNote>
                                 <decoNote xml:id="b3" type="SewingStations">3</decoNote>
                                 <decoNote xml:id="b4">
-                                  Chainstitch sewing on three sewing stations; continuous fold pattern; S-plied thread.
-                                  The bookblock was probably sewn on four sewing stations and has been resewn on three, also substituting the boards. On the folios, the holes and the marks left by the previous sewing are present.
-                                  The ends of the thread are knotted in the centrefold of the first quire.
-                                  <locus target="#53"/> is stitched through the inner margin directly to <locus target="#54"/>.
+                                    Chainstitch sewing on three sewing stations; continuous fold pattern; S-plied thread.
+                                    The bookblock was probably sewn on four sewing stations and has been resewn on three, also substituting the boards. On the folios, the holes and the marks left by the previous sewing are present.
+                                    The ends of the thread are knotted in the centrefold of the first quire.
+                                    <locus target="#53"/> is stitched through the inner margin directly to <locus target="#54"/>.
                                 </decoNote>
                                 <decoNote xml:id="b5" type="Spine">
-                                  A piece of folded parchment protects the spine and is linked to the bookblock by stitching it to the holes for board attachment and the chains that the sewing forms on the spine.
-                                  A yellow thread emerges from the tail edge of the bookblock.
+                                    A piece of folded parchment protects the spine and is linked to the bookblock by stitching it to the holes for board attachment and the chains that the sewing forms on the spine.
+                                    A yellow thread emerges from the tail edge of the bookblock.
                                 </decoNote>
                             </binding>
                         </bindingDesc>
-
+                        
                     </physDesc>
 
-                <!--    <history>
-                        <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
+                    <history>
+                        <provenance>Unknown provenance. According to <ref target="#e1">Additional note 1</ref>, the manuscript was
```

Comment:
e1 is technically not "additional note 1" (that would be a1). Maybe "Varia note 1"?